src/components/Dashboard/PatientCard/index.js

```diff
@@ -133,41 +133,47 @@ const PatientCard = (props) => {
       <div>
         {patient && patient.vitals && (
           <div className="body-vitals-status-wrap">
-            {Object.entries(patient.vitals).map((bodyParms, i) => {
+            {Object.entries(patient.vitals).map((bodyParams, i) => {
+              if (!bodyParams[1].data.length) {
+                return null;
+              }
+
               return (
                 <div className="content-wrp" key={i}>
```

Comment:
please do not use index as a key https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9 , getRandomKey is better if there is no unique data field you can use instead of index